textDisplay.cc

```diff
@@ -0,0 +1,82 @@
+#include "textdisplay.h"
+
+TextDisplay::TextDisplay(int n):View(n){
+	// Creates a new nxn textdisplay and sets the default value to '0'
+	theDisplay = new char*[n];
+	for (int row = 0; row < n; row++){
```

Comment:
Do you mean initially create a blank board with just "black" and "white" squares, and then notify the view for the initial position for each piece? I like that idea, cause that might be useful for the implementation of Chess960